Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
Please add a TODO/TBD here mentioning that this file is not to be commited. 
OR
 move this to test folder; so that it can be used later also

Reply:
Its at the root of the Repo..So, this will not be installed as part of RPM. So, I think we can keep it here